src/app/app.routes.ts

```diff
@@ -1,6 +1,6 @@
 import { Routes } from '@angular/router';
 import { RolesEnum, RoutesUrlsEnum } from '@common/enums';
-import { authGuard, profileCompletedGuard, roleGuard } from '@core/guards';
+import { authGuard, profileCompletedGuard, roleGuard,forgotPasswordGuard  } from '@core/guards';
```

Comment:
Ajuste esto aqui que haya espacios entre las importaciones.


```suggestion
import { authGuard, profileCompletedGuard, roleGuard, forgotPasswordGuard  } from '@core/guards';
```